### PR TITLE
/element/location

### PR DESCRIPTION
**Pull request by Jonahss**
submitting this PR in favor of #52

### CODE REVIEW

File: WebDriverAgentLib/Categories/UIAElement+WebDriverAttributes.m [hunk 1]
```diff
@@ -46,6 +46,15 @@ - (NSDictionary *)wdSize
       };
 }
 
+- (NSDictionary *)wdLocation
+{
+    return
```
Comment (marekcirkos) [inline WebDriverAgentLib/Categories/UIAElement+WebDriverAttributes.m]: You could use `wdRect` in those as well when fixing build issue.

**Comment by Jonahss** [comment]
will do.

**Comment by Jonahss** [comment]
@marekcirkos fixed

**Comment by marekcirkos** [comment]
Awesome! Thank you

**Pull request merged**